backend/src/controllers/manager/assignEmployeeCourses.ts

```diff
@@ -64,6 +72,23 @@ const assignEmployeeCourses = async (
       }),
     );
 
+    const userFullName = combineFullName(employee.firstName, employee.lastName);
+
+    assignedCourses.map(async (clientCourse) => {
+      if (clientCourse) {
+        const course = await getCourseProvider(clientCourse.course, employeeId);
+        await addUserNotification(
+          employeeId,
+          course.title,
+          userFullName,
+          NotificationStatuses.new,
+          NotificationTitles.assigned,
+          NotificationDescription.assigned,
+          NotificationType.employee,
+        );
+      }
+    });
+
```

Comment:
We already have `addNotification` separate controller in our sequence. For what reason you're duplicating this code?